libs/blocks/global-navigation/global-navigation.js

```diff
@@ -109,6 +117,16 @@ const decorateProfileTrigger = async ({ avatar }) => {
   return buttonElem;
 };
 
+let keyboardNav;
+const setupKeyboardNav = async () => {
+  if (keyboardNav) return;
+  keyboardNav = keyboardNav || new Promise(async (resolve) => {
```

Comment:
If `keyboardNav` exists, then the method returns immediately, so I'm not sure whether the 'or' condition is needed here.